### PR TITLE
Please the architect god

### PR DESCRIPTION
**Pull request by sslavic**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by sslavic** [comment]
Going with a ping

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added GitHub workflows.
 
-[Unreleased]: https://github.com/giantswarm/cluster-operator/tree/legacy
+[Unreleased]: https://github.com/giantswarm/cluster-operator/tree/master
```
Comment (xh3b4sd) [inline CHANGELOG.md:14]: @kopiczko what are your thoughts on this? The change does not look right but it was necessary to keep moving.

Comment (kopiczko) [reply CHANGELOG.md:14]: That's a bug. Needs to be fixed.

Comment (kopiczko) [reply CHANGELOG.md:14]: Okay I spend some more time here and I don't think fix for this is worth the effort. We should always start with changelogs in the master branch. We'll have a few outliers like this one here but I don't think it's a big issue.

Comment (kopiczko) [reply CHANGELOG.md:14]: I'm doing this anyway. I'll need to release architect for my other work.

**Pull request merged**